
Beurh I know don't ask me why it's needed :-( But since the move to
kernel-headers the include was broken.

--- glibc-2.2.4/malloc/memusagestat.c.chmou	Mon Jul  9 20:57:38 2001
+++ glibc-2.2.4/malloc/memusagestat.c	Wed Nov 21 11:12:04 2001
@@ -33,9 +33,9 @@
 #include <sys/param.h>
 #include <sys/stat.h>
 
-#include <gd.h>
-#include <gdfontl.h>
-#include <gdfonts.h>
+#include "/usr/include/gd.h"
+#include "/usr/include/gdfontl.h"
+#include "/usr/include/gdfonts.h"
 
 
 /* Default size of the generated image.  */
